who/templates/core/tags/footerpage.html

```diff
@@ -1,20 +1,15 @@
-{% load wagtailcore_tags wagtailimages_tags core_tags wagtailsettings_tags %}
+{% load wagtailcore_tags core_tags wagtailsettings_tags %}
 {% get_settings %}
 {% if settings.profiles.UserProfilesSettings.terms_and_conditions %}
   {% pageurl settings.profiles.UserProfilesSettings.terms_and_conditions as terms_url %}
 {% endif %}
-{% social_media_footer %}
+
```

Comment:
is the removal of the tag `{% social_media_footer %}` an intentional change?

Reply:
@moh-moola It's added on the base file instead. Thanks